packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -199,6 +249,89 @@ function renderWithFalsyKey(props = {}) {
+          </Text>
+        </Content>
+      </Item>
     </Tabs>
   );
 }
+
+let ManyTabs = (props = {}) => {
+  let items = [
+    {name: 'Tab 1', children: 'Tab Body 1', icon: <Dashboard size="S" />},
+    {name: 'Tab 2', children: 'Tab Body 2', icon: <Calendar size="S" />},
+    {name: 'Tab 3', children: 'Tab Body 3', icon: <Bookmark size="S" />},
+    {name: 'Tab 4', children: 'Tab Body 4', icon: <Dashboard size="S" />},
+    {name: 'Tab 5', children: 'Tab Body 5', icon: <Calendar size="S" />},
+    {name: 'Tab 6', children: 'Tab Body 6', icon: <Bookmark size="S" />}
+  ];
+
+  let [tabs, setTabs] = React.useState(items);
+  let addTab = () => {
+    let newTabs = [...tabs];
+    newTabs.push({
+      name: `Tab ${tabs.length + 1}`,
+      children: `Tab Body ${tabs.length + 1}`
+    });
+
+    setTabs(newTabs);
+  };
+
+  let removeTab = () => {
+    let newTabs = [...tabs];
+    newTabs.pop();
+    setTabs(newTabs);
+  };
+
+  return (
+    <div style={{width: '80%'}}>
+      <Tabs {...props} aria-label="Tab example" items={tabs} onSelectionChange={action('onSelectionChange')}>
+        {item => (
+          <Item
+            key={item.name}
+            textValue={item.name}
+            title={
+              <>
+                {item.icon}
+                <Text>{item.name}</Text>
+              </>
+            }>
+            <Content margin="size-160">
+              <Heading>{item.children}</Heading>
+              <Text>
+                Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+                Id cupidatat aute id ut excepteur exercitation magna pariatur. Mollit irure irure reprehenderit pariatur eiusmod proident Lorem deserunt duis cillum mollit. Do reprehenderit sit cupidatat quis laborum in do culpa nisi ipsum. Velit aliquip commodo ea ipsum incididunt culpa nostrud deserunt incididunt exercitation. In quis proident sit ad dolore tempor. Eiusmod pariatur quis commodo labore cupidatat cillum enim eiusmod voluptate laborum culpa. Laborum cupidatat incididunt velit voluptate incididunt occaecat quis do.
+                Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
+              </Text>
+            </Content>
+          </Item>
+        )}
+      </Tabs>
+      <ButtonGroup marginEnd="30px">
+        <Button variant="secondary" onPress={() => addTab()}>
+          <Text>Add Tab</Text>
+        </Button>
+        <Button variant="secondary" onPress={() => removeTab()}>
```

Comment:
So, if you click the remove button until there are no tabs this breaks. I'm not sure that is a real error that would be coded. Maybe someone creating the tabs dynamically and the API returns no tabs and the component breaks instead of loading nothing. Should we create a no tabs in a tablist story or just prevent this from deleting the last tab?

```TypeError: Cannot read property 'getBoundingClientRect' of undefined
    at getFurthestPoint ```

Reply:
Good catch, I'll make it so it doesn't break and leave it to the user to remove the component if their list of tabs is empty.